account/lnurl.go

```diff
@@ -62,7 +62,61 @@ func (a *Service) HandleLNURL(rawString string) (result *data.LNUrlResponse, err
 
 	rawurl, iparams, err := lnurl.HandleLNURL(encodedLnurl)
 	if err != nil {
-		return nil, fmt.Errorf("%v : %w", handleLNUrlError, err)
+		if lightningAddress != "" {
+
+			const unknownResponseTagError = "unknown response tag "
+			if strings.HasPrefix(err.Error(), unknownResponseTagError) {
```

Comment:
Is all this only to show a more accurate error? I am not sure it worth it and definitely doesn't justify an extra http request.

Reply:
Yes, that's the only reason. I'm happy to delete it though.

Reply:
Let's simply as Roei suggests.